### PR TITLE
August Heartbeat Blog post

### PR DESCRIPTION
**Pull request by jendefig**
Projected Publish Date: August 16, 2022

Checklist: 
- [ ] There are no grammatical errors, jargon, or obvious stylistic issues. (**for reviewer to check**)
- [x] Have images been run through tiny.png?
- [x] Cover image complete
- [x] Are all lines 80 characters or less?
- [x] Are Oxford commas observed?
- [x] Content Creator has done a complete review for typos/language after taking a break from content
- [x] links checked in the piece
- [x] Have reviewed what the piece looks like on your own [[machine using yarn develop](https://dvc.org/doc/user-guide/contributing/docs#development-environment)](https://dvc.org/doc/user-guide/contributing/docs#development-environment)
- [x] changed discuss link
- [x] Tweet loves are public tweets

### CODE REVIEW

**Comment by jendefig** [comment]
@rogermparent I've checked all the failed Twitter links and they work in an incoginito window.  Correcting the bad github one now...

**Comment by rogermparent** [comment]
Ah, yeah, sorry about that. Twitter seems to be blocking our checks, escalating the arms race between our link checker and the checked websites.

Unfortunately, for now we'll just have to ignore the twitter links and hope they're good. I'll try and probe into why they started failing and fix it on the link check end!

**Review by SoyGema — COMMENTED** [review]
**Overall Feedback :**

- Good Timed cognitive attention loop for each part 
- Good use of figures separating content 
- Overall  adding value content 

**Minor changes request :** 

- Typo in Gema´s Last Name ( is Gema P**a**rreño ) in some [ lines ](https://github.com/iterative/dvc.org/compare/main...hb-august#diff-f7ebc75d2388d87477b51486fe4fac8f035f1829c4e637e38638bfe8c109cd96R206) and in hiring lines as well 
I would change the ¡Viva España! expression , as it might been associated to a particular political branch [here](https://github.com/iterative/dvc.org/compare/main...hb-august#diff-f7ebc75d2388d87477b51486fe4fac8f035f1829c4e637e38638bfe8c109cd96R265) Some suggestions : happy to see our Spanish team growing ! Spain is growing strong at Iterative! 

Congratulations for the piece!

File: content/blog/2022-08-16-august-22-heartbeat.md [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+perspectives from different people across all parts of the data space, talking
+about challenges, danger zones, and issues that we all need to be aware of in
+the field. Recently Hugo started a new podcast,
+[Vanishing Gradients](https://vanishinggradients.fireside.fm/). This newer
+endeavor is in a somewhat different format than DataFramed, but still with
+Hugo's characteristic deep dive into all the challenges that come up when
+working with data. Hugo uses a long-format conversation approach with many
+leaders and great thinkers in the data science/machine learning/AI space. In
+episodes, [seven](https://vanishinggradients.fireside.fm/7) and
+[eight](https://vanishinggradients.fireside.fm/8) Hugo has a fascinating chat
```
Comment (flippedcoder) [inline content/blog/2022-08-16-august-22-heartbeat.md:72]:
```suggestion
episodes [seven](https://vanishinggradients.fireside.fm/7) and
[eight](https://vanishinggradients.fireside.fm/8), Hugo has a fascinating chat
```

File: content/blog/2022-08-16-august-22-heartbeat.md [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+experiment features in both tools in the coming months! Also, experimenting with
+DVC in Kaggle may give you some good practice for things we are cooking up
+internally! 😉🤫
+
+## **Shambhavi Mishra** - Searching for Semantic Similarity
+
+[**Shambhavi Mishra**](https://twitter.com/ShambhaviCodes) in her post
+[Searching for Semantic Similarity](https://medium.com/towards-artificial-intelligence/searching-for-semantic-similarity-cfbff2388d04)
+details the steps of her NLP project on similarity algorithms. She mainly
+focuses on cosine similarity using a Stackoverflow questions dataset. The
```
Comment (flippedcoder) [inline content/blog/2022-08-16-august-22-heartbeat.md:116]:
```suggestion
focuses on cosine similarity using a Stack Overflow questions dataset. The
```

File: content/blog/2022-08-16-august-22-heartbeat.md [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+to mock the S3 for the data. Then he shows you how to link the MinIO server as
+DVC remote storage.
+
+![Minio Browser](/uploads/images/2022-08-16/minio.png '=800') _Minio Browser
+with Data pushed from DVC
+([Source link](https://betterprogramming.pub/run-s3-locally-with-minio-for-dvc-machine-learning-pipeline-7fa3d240d3ab))_
+
+## **Caleb Kaiser** - Moving from Data Science to Machine Learning Engineering
+
+It can sometimes be confusing to determine where data science stops and machine
```
Comment (flippedcoder) [inline content/blog/2022-08-16-august-22-heartbeat.md:146]: Could you break this up into a couple of paragraphs? It'll break up the wall of text.

File: content/blog/2022-08-16-august-22-heartbeat.md [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+one of the examples and shows the promise in some tools that bridge the gap
+between machine learning and software engineering where he highlights DVC and
+Huggingface. This is a good piece to read if you are grappling with the
+difference!
+
+https://media.giphy.com/media/xUNd9DLukkavmhybAs/giphy.gif
+
+## Just a few other things...
+
+- Github Goodness alert for
```
Comment (flippedcoder) [inline content/blog/2022-08-16-august-22-heartbeat.md:164]:
```suggestion
- GitHub Goodness alert for
```

Double-check the spelling throughout.

File: content/blog/2022-08-16-august-22-heartbeat.md [hunk 5]
```diff
@@ -0,0 +1,360 @@
...
+CLI, API, or web UI. Be sure to try it out and give us your feedback. Learn more
+[in the blog post](https://dvc.org/blog/iterative-studio-model-registry) and
+[in the docs](https://dvc.org/doc/studio/user-guide/model-registry/what-is-a-model-registry).
+Look out for a full tutorial coming soon!
+
+https://www.youtube.com/watch?v=DYeVI-QrHGI
+
+### 🧑🏽‍💻 Iterative's First Internal Hackathon
+
+Last week we had our very first internal Hackathon! The entire company
```
Comment (flippedcoder) [inline content/blog/2022-08-16-august-22-heartbeat.md:199]: Break this up into a couple of paragraphs.

**Comment by jendefig** [comment]
@flippedcoder and @SoyGema Have made the changes.  Just need one approval to merge.

**Review by SoyGema — APPROVED** [review]

**Pull request merged**